spec/unit/resource/api300/c7000/enclosure_group_spec.rb

```diff
@@ -45,4 +48,53 @@
       expect(item.set_script('Blah')).to eq(true)
     end
   end
+
+  describe '#create_interconnect_bay_mapping' do
+    it 'creates entries for each bay in each enclosure' do
```

Comment:
@tmiotto , this is the test that proves the mapping gets created on initialization.